bft_types/src/lib.rs

```diff
@@ -199,10 +199,15 @@ impl BfProgram {
     }
 
     /// The instructions that make up this program
-    pub fn instructions(&self) -> &[LocalisedInstruction] {
+    pub fn localised_instructions(&self) -> &[LocalisedInstruction] {
         &self.instructions
     }
 
+    /// The vector of indexes '[' and ']' may jump to
+    pub fn jump_map(&self) -> &Vec<Option<usize>> {
```

Comment:
I don't like this function signature, I'd suggest:

`pub fn jump_target(&self, ip: usize) -> usize`